### ISSUE TITLE
Symbol Operators (e.g. descending) not working

### ISSUE THREAD

**Issue by jhilden** [Fosome/garb#96]
For some reason the SymbolOperators (desc, eql, ...) in Garb are not working for me.  
This is my code:

```
class SearchTerms
  extend Garb::Model
  metrics :search_result_views
  dimensions :search_keyword
end

SearchTerms.results(profile, :limit => 50, :sort => :search_result_views.descending)
```

I get the following error:

```
undefined method `descending' for :search_result_views:Symbol
```

It happens within a rails 3.0.9 app using garb 0.9.1.  Garb is normally loaded in the Gemfile and all the other Garb functionality works as expected.

Does anybody have an idea why the SymbolOperators (from garb/lib/support.rb) are not proberly loaded?  Any help would be greatly appreciated.

**Comment by tpitale** [comment]
Are you using DataMapper, MongoMapper or some other ORM that uses symbol keys? This is almost certainly the issue.

I'm working on a branch that removes the symbol operator usage since it constantly conflicts with other gems. We'll be moving to a named filters behavior that turns out to be more expressive and powerful.

**Comment by tpitale** [comment]
Anything, @jhilden?

**Comment by jhilden** [comment]
Sorry for not getting back.  Thanks for rechecking on my problem.

We are using plain ActiveRecord, so an ORM can't be the issue.  Maybe some other gem, but I wouldn't know where to start searching.

I just switched from the released version 0.9.1 to the current master and that worked for me.  Therefore I guess I will just close this ticket.
